logbook-jaxws/pom.xml

```diff
@@ -0,0 +1,39 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.zalando</groupId>
+		<artifactId>logbook-parent</artifactId>
+		<version>2.5.0-SNAPSHOT</version>
+	</parent>
+	<artifactId>logbook-jaxws</artifactId>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.zalando</groupId>
+			<artifactId>logbook-api</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.zalando</groupId>
+			<artifactId>logbook-core</artifactId>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>jakarta.xml.ws</groupId>
```

Comment:
Wouldn't this be better? https://search.maven.org/artifact/javax.xml.ws/jaxws-api

Reply:
I think not. Oracle decided to transfer Java EE fully to an open source foundation (Eclipse). If you open the home page in the link that you shared you can see that information

Reply:
Ah, good to know!